e2e-tests/smart-update2/run

```diff
@@ -76,14 +79,143 @@ function deploy_version_service {
 	sleep 10
 }
 
+function check_telemetry_transfer() {
+	local cr_vs_uri=${1}
+	local cr_vs_channel=${2:-"disabled"}
+	local telemetry_state=${3:-"enabled"}
+
+	desc 'create PXC minimal cluster'
+	cluster="minimal-cluster"
+
+	kubectl_bin apply -f "${conf_dir}/client.yml"
+	yq "${conf_dir}/secrets.yml" \
+		| yq eval "(. | select(.metadata.name == \"my-cluster-secrets\") | .metadata.name) = \"${cluster}\"" \
+		| kubectl_bin apply -f -
+
+	yq "${src_dir}/deploy/cr-minimal.yaml" \
+		| yq eval ".spec.upgradeOptions.versionServiceEndpoint=\"${cr_vs_uri}\"" \
+		| yq eval ".spec.upgradeOptions.apply=\"${cr_vs_channel}\"" \
+		| yq eval ".spec.initImage=\"$IMAGE\"" \
+		| yq eval '.spec.crVersion="9.9.9"' \
+		| yq eval ".spec.pxc.image=\"$IMAGE_PXC\"" \
+		| yq eval ".spec.haproxy.image=\"$IMAGE_HAPROXY\"" \
+		| yq eval ".spec.logcollector.image=\"$IMAGE_LOGCOLLECTOR\"" \
+		| kubectl_bin apply -f -
+
+	desc 'check if Pod is started'
+	wait_for_running "$cluster-pxc" "1"
+	sleep 20
+	local proxy
+	proxy=$(get_proxy "$cluster")
+	wait_for_running "$proxy" 1
+
+	desc 'write data'
+	if [[ $IMAGE_PXC =~ 5\.7 ]] && [[ "$(is_keyring_plugin_in_use "$cluster")" ]]; then
+		encrypt='ENCRYPTION=\"Y\"'
+	fi
+	run_mysql \
+		"CREATE DATABASE IF NOT EXISTS myApp; use myApp; CREATE TABLE IF NOT EXISTS myApp (id int PRIMARY KEY) $encrypt;" \
+		"-h $proxy -uroot -proot_password -P$port"
+	run_mysql \
+		'INSERT myApp.myApp (id) VALUES (100500)' \
+		"-h $proxy -uroot -proot_password -P$port"
+
+	kubectl_bin logs "$(kubectl get pods --selector=run=version-service-cr -o jsonpath='{.items[0].metadata.name}')" \
+		| grep -E 'server request payload|unary call' \
+		| grep -Eo '\{.*\}' \
+		| jq 'del(."grpc.request.content".msg.customResourceUid)' \
+		| jq 'del(."grpc.request.content".msg.kubeVersion)' \
+		| jq 'del(."grpc.start_time")' \
+		| jq 'del(."grpc.time_ms")' \
+			>"${tmp_dir}/${telemetry_state}_telemetry.version-service-cr.log.json"
+
+	kubectl_bin logs "$(kubectl get pods --selector=run=version-service -o jsonpath='{.items[0].metadata.name}')" \
+		| grep -E 'server request payload|unary call' \
+		| grep -Eo '\{.*\}' \
+		| jq 'del(."grpc.request.content".msg.customResourceUid)' \
+		| jq 'del(."grpc.request.content".msg.kubeVersion)' \
+		| jq 'del(."grpc.start_time")' \
+		| jq 'del(."grpc.time_ms")' \
+			>"${tmp_dir}/${telemetry_state}_telemetry.version-service.log.json"
+
+	local telemetry_log_file="${telemetry_state}_telemetry.version-service${OPERATOR_NS:+-cw}.log.json"
```

Comment:
Maybe I'm missing it, but in case of cluster wide I don't see where we create this file?

Reply:
I think the same for ${telemetry_state}_telemetry.version-service-cr.log.json

Reply:
@tplavcic I think it is for compare file please see line 145

Reply:
@tplavcic I think it is for compare file please see line 145